lms/djangoapps/instructor_task/api.py

```diff
@@ -250,3 +253,15 @@ def submit_cohort_students(request, course_key, file_name):
     task_key = ""
 
     return submit_task(request, task_type, task_class, course_key, task_input, task_key)
+
+
+def submit_ora2_request_task(request, course_key):
+    """
+    AlreadyRunningError is raised if an ora2 report is already being generated.
+    """
+    task_type = 'ora2_responses'
```

Comment:
And for consistency we can name the function `submit_export_ora2_data` and the `task_type` `export_ora2_data`.